jobs/consul_agent_windows/templates/pre-start.ps1.erb

```diff
@@ -0,0 +1,10 @@
+$LOG_DIR="/var/vcap/sys/log/consul_agent_windows"
+$DATA_DIR="/var/vcap/data/consul_agent_windows"
+$CONF_DIR="/var/vcap/jobs/consul_agent_windows/config"
+$CERT_DIR="$CONF_DIR/certs"
+
+New-Item -ItemType Directory -Force -Path "${LOG_DIR}"
+New-Item -ItemType Directory -Force -Path "${DATA_DIR}"
+New-Item -ItemType Directory -Force -Path "${CONF_DIR}"
+New-Item -ItemType Directory -Force -Path "${CERT_DIR}"
+
```

Comment:
the non-windows version of pre-start modifies `/etcd/resolvconf/resolv.conf.d/head` so that consul domain DNS queries go to the consul agent (via dnsmasq).  How does that work in this world?